inspirehep/modules/records/jsonschemas/records/elements/titles.json

```diff
@@ -0,0 +1,21 @@
+{
+    "titles": {
```

Comment:
@jacquerie as discussed IRL we should remove this `titles` attribute, thus de-indenting everything by one level and adding the appropriate `$schema` entry.